src/llmcompressor/modifiers/transform/spinquant/base.py

```diff
@@ -121,6 +121,9 @@ def validate_rotations(cls, value):
         return value
 
     def on_initialize(self, state: State, **kwargs) -> bool:
+        from llmcompressor.transformers.compression.compressed_tensors_utils import (
+            untie_word_embeddings,
```

Comment:
I'm surprised that you need to dynamically import this. Maybe we can move this CT in the future.